webapp/src/components/App/Singlestudy/explore/Debug/utils.ts

```diff
@@ -49,17 +50,37 @@ export interface DataCompProps extends FileInfo {
   reloadTreeData: () => void;
 }
 
+interface ContentParsingOptions {
+  filePath: string;
+  fileType: string;
+}
+
 ////////////////////////////////////////////////////////////////
-// File Info
+// Utils
 ////////////////////////////////////////////////////////////////
 
+const URL_SCHEMES = {
+  MATRIX: ["matrix://", "matrixfile://"],
```

Comment:
For this type of constant object where both the object name and its keys represent fixed values or schemes, it's more conventional to have both in UPPER_SNAKE_CASE, it's not a regular enum-like object see: https://google.github.io/styleguide/tsguide.html#naming

Reply:
I know its confusing, the style guide example uses lowercase because they're dealing with actual unit names rather than symbolic constants.

The difference is that:

In URL_SCHEMES, the keys (MATRIX, JSON, FILE) are symbolic identifiers representing categories/types
In UNIT_SUFFIXES, the keys ('milliseconds', 'seconds') are the actual string names of time units

look at "Rules by identifier type" section:
says:  "CONSTANT_CASE 	global constant values, including enum values. See [Constants](https://google.github.io/styleguide/tsguide.html#identifiers-constants) below."

Reply:
we should create a ticket and spend time to make our set of rules, especially for this kind of conventions

Reply:
for a more general example:

1. When keys are symbolic constants (use UPPER_SNAKE_CASE):
```typescript
const ERROR_CODES = {
    NOT_FOUND: 404,
    SERVER_ERROR: 500,
    UNAUTHORIZED: 401
};
```
Here, NOT_FOUND is a symbolic name representing the concept of a "not found" error. The key itself is a symbol/identifier.

2. When keys are actual VALUES/NAMES (use lowercase):
```typescript
const MIME_TYPES = {
    'image/jpeg': '.jpg',
    'image/png': '.png',
    'text/html': '.html'
};
```
Here, 'image/jpeg' is the actual MIME type string that would be used in HTTP headers.


Think of it this way:
- If you're using the key as a reference/symbol (like URL_SCHEMES.MATRIX), use UPPER_SNAKE_CASE
- If you're using the key as a literal value (like MIME_TYPES['image/jpeg']), use lowercase

Reply:
Alright! I didn't know this subtlety.